components/list/list.css

```diff
@@ -0,0 +1,43 @@
+.ordered-list {
+	list-style-type: lower-alpha;
+	max-width: 15rem;
+}
+
+.unordered-list {
+	list-style-type: circle;
+	max-width: 15rem;
+}
+
+.image-list {
+	list-style-type: "🎲";
+	max-width: 15rem;
+}
+
+.notype-list {
+	list-style-type: none;
+	max-width: 15rem;
+}
+
+.ordered-list .list-title,
+.ordered-list li,
+.ordered-list .list-subtitle,
+.unordered-list .list-title,
+.unordered-list li,
+.unordered-list .list-subtitle,
+.image-list .list-title,
+.image-list li,
+.image-list .list-subtitle,
+.notype-list .list-title,
+.notype-list li,
+.notype-list .list-subtitle {
```

Comment:
It's better to use a class for padding here.

Reply:
The motive for this was so that the user wont have to put another class along with the list class, but will think of a better way how this can be done next time, thank you!